mantidproject/mantid
# LoadPSIMuonBin Muon GUI Integration

**Description of work.**
Alongside my work on #25684, I worked on making sure the Algorithm loaded into the GUI correctly which goes hand in hand with all the changes on this PR. Allowing the user to take full advantage of most of the GUI's features.

**Report to:** Francis Pratt (ISIS)

**To test:**
- Open `Interfaces->Muon->Frequency Domain Analysis`
- Click Browse near the top of the window
- Browse to your Build/Testing/Data/UnitTest/deltat_tdc_dolly_1529.bin and load it
- Check that the Grouping Table has done something sensible
- Then switch to the Phase Table tab
  - Set First Good Data to 0.17 and Last Good Data to 10.0
  - Calculate Phase Table
  - Calculate Phase Quad
  - Both should happen without an issue
- Switch to the transform tab
  - Calculate FFT
  - Switch FFT to MaxEnt
  - Untick Fit dead times
  - Calculate MaxEnt
- Have a look at the data and make sure it has done something sensible.





Fixes #23070 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
